### ISSUE TITLE
Twitter cards not working for English lessons

### ISSUE THREAD

**Issue by wcaleb** [programminghistorian/jekyll#935]
I noticed today that Twitter cards for English lessons are not working (see #402 for discussion of Twitter cards). It looks like maybe something in https://github.com/programminghistorian/jekyll/commit/c4ee8761d26c14fedb3f9a7a380985d0d5bd3df8 is creating double backslashes in the URL to the image needed to generate the card.

**Comment by mdlincoln** [comment]
Thanks for the heads up - I'll take a look.

**Comment by walshbr** [comment]
@mdlincoln should line 4 of _includes/twitter-card.html just be

`<meta name="twitter:image" content="{{ site.redirect_from }}/gallery/{{ page.url | remove_first:'/lessons/' }}.png" />`

instead of

`<meta name="twitter:image" content="{{ site.liveurl }}/gallery/{{ page.url | remove_first:'/lessons/' }}.png" />`

That seems to copy the es logic, at least, and i looks to construct the URLs correctly. But I'm not as familiar with that include or the metadata as you are.

**Comment by acrymble** [comment]
we're still tweeting in English. Has this been fixed?

**Comment by walshbr** [comment]
Not fixed @acrymble - the tweets still work regardless of whether or not we have working twitter cards. I was waiting for feedback before implementing that line. I'm less familiar with the way the es/en sites are all linked up, so I was reluctant to go about messing with that side of the site. But if @mdlincoln thinks it makes sense it's an easy fix to implement.

**Comment by walshbr** [comment]
I noticed this still wasn't working. I think this might be because we were still linking out to lesson links w/o the /en/lessons construction (since we populated that spreadsheet a long time ago). Working theory is that the cards weren't working with the redirects that we have in place. I updated the spreadsheet with the new /en/ links, and I'll close once it's clear that this is working. Should just be a matter of waiting for the bot to tweet an English lesson out in the next couple days.
